README.md

```diff
@@ -298,11 +299,77 @@ By specifying the `contractname`, Timber will know: which contract to interact w
 E.g. `"contractname": "MerkleTreeControllerSHA"`
```

Comment:
Would it be possible to also move `contractname` from the header to the body?
Since the body doesn't need all lower-case, it should then be updated to `contractName`.
Similarly we should probably update it to `treeId` (rather than `treeid`).

Reply:
Added in 0d6a7f9 :)